lib/src/common.dart

```diff
@@ -24,7 +24,7 @@ class Common {
 
         int last = bindings.obx_last_error_code();
         int last2 = bindings.obx_last_error_secondary();
-        String desc = CString.fromPtr(bindings.obx_last_error_message()).val;
+        String desc = Utf8.fromUtf8(bindings.obx_last_error_message().cast<Utf8>());
```

Comment:
same as above, the cast wouldn't be necessary

Reply:
```suggestion
        String desc = Utf8.fromUtf8(bindings.obx_last_error_message());
```

Reply:
idem dito, unless I overlooked something